CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 2.8.0 (2020-04-17)
+### AWSXRayRecorder.Core (2.8.0)
+#### Fixed
+- Fixed customer start/end timestamps floor [PR#119](https://github.com/aws/aws-xray-sdk-dotnet/pull/119)
+- Added DelegateExporter to JasonSegmentMarshaller [PR#122](https://github.com/aws/aws-xray-sdk-dotnet/pull/122)
+
+### AWSXRayRecorder.Handlers.AwsSdk (2.8.0)
+#### Improved
+- Added Whitelisting EndpointName parameter for InvokeEndpoint operation for SageMakerRuntime [PR#117](https://github.com/aws/aws-xray-sdk-dotnet/pull/117)
+
+### AWSXRayRecorder.Handlers.EntityFramework (1.0.0)
+#### New Feature
+- Added tracing support for Entity Framework Core 3.0 and above [PR#124](https://github.com/aws/aws-xray-sdk-dotnet/pull/124)
+- Modified README.md and removed some code comments [PR#127](https://github.com/aws/aws-xray-sdk-dotnet/pull/127)
```

Comment:
it's okay if we don't include README changes. should highlight more bugs, breaking changes, new features and improvements.